QA.md

```diff
@@ -83,6 +83,11 @@
     1. Generate a payment request of 2 TEST.
     1. Pay the Payment Request using your test wallet that has just received the TEST before and check the confirmation message.
 
+1. **Send HTR with QR Code**
+    1. Generate a payment request requesting 1 HTR from another device
```

Comment:
Does it make sense to generate a payment request of 1,000,000 HTR, so we can have an error handling of no funds? Then create this normal payment request of 1 and continue as you detailed?

Reply:
For the current QR Code reading flow, the lack of funds is only validated when the actual `tx` is being built, because no utxos are found to complete its inputs. ( see screenshot below )


Maybe it would be better to add this step when we introduce a dedicated validation on the interface, such as the one we have with the manual inputs. What do you think?

![No funds Screenshot](https://github.com/HathorNetwork/hathor-wallet-mobile/assets/1299409/0ff4f7ad-0461-46de-b0b4-64f06a217697)